R/fwrite.R

```diff
@@ -115,3 +115,5 @@ fwrite = function(x, file="", append=FALSE, quote="auto",
   invisible()
 }
 
+nozlib = function() identical(.Call(Cdt_zlib_version), "zlib header files were not found when data.table was compiled")
```

Comment:
NB: this may fail under Chinese (or other locales eventually) session since this string is passed through `gettext()`.

https://github.com/Rdatatable/data.table/blob/50a3dc3744a8039b2215ad6a348bf2aea1e5b3d7/src/utils.c#L376-L378

Safer to test on the `NOZLIB` branch, possibly on `grepl("ZLIB_VERSION", ., fixed = TRUE)` in case that message is later marked for translation?

Alternatively we could make a companion function to `dt_zlib_version` like `dt_has_zlib` that just returns `TRUE`/`FALSE`.